import_export/forms.py

```diff
@@ -41,6 +41,13 @@ def __init__(self, import_formats, *args, **kwargs):
             choices.insert(0, ("", "---"))
```

Comment:
You're right.  I looked into this and decided that the forms module could do with a refactor to avoid duplication.  This means that I have added a few more changes so that the forms classes are standardised.

I'd be grateful if you could take a look.